frontends/main/src/test-utils/index.tsx

```diff
@@ -124,7 +124,7 @@ const expectLastProps = (
 ) => {
   expect(fc).toHaveBeenLastCalledWith(
     expect.objectContaining(partialProps),
-    expect.anything(),
+    expect.toBeOneOf([expect.anything(), undefined]),
```

Comment:
Evidently how react calls functional components changed a little between react 18 -> 19.

It used to call them as `func(props, something_i_didnt_understand)`, now it calls them as `func(props, undefined)`.  This is irrelevant and is is already what's happening in MIT Learn on RC/Prod.